edgelet/build/windows/openssl.ps1

```diff
@@ -8,40 +8,52 @@ function Get-OpenSSL
 
     $ErrorActionPreference = 'Continue'
 
-    if (!((Test-Path -Path $env:HOMEDRIVE\vcpkg) -and ((Test-Path -Path $env:HOMEDRIVE\vcpkg\vcpkg.exe))))
+    # Override vcpkg directory from the default $env:HOMEDRIVE\vcpkg that VSTS CI machines have.
+    # See TODO below for the reason.
+    $vcpkgDirectory = "$env:HOMEDRIVE\vcpkg-2"
+
+    if (!((Test-Path -Path $vcpkgDirectory) -and ((Test-Path -Path $vcpkgDirectory\vcpkg.exe))))
     {
         Write-Host "Installing vcpkg from github..."
-        git clone https://github.com/Microsoft/vcpkg $env:HOMEDRIVE\vcpkg
+        git clone https://github.com/Microsoft/vcpkg $vcpkgDirectory
         if ($LastExitCode)
         {
             Throw "Failed to clone vcpkg repo with exit code $LastExitCode"
         }
+
+        # TODO: vcpkg updated openssl from 1.0.2 to 1.1.1, but some of our libiothsm tests fail with it.
+        #       Revert to the version of vcpkg before that commit to fix our build for now,
+        #       until we fix the tests to work with 1.1.1.
+        Push-Location $vcpkgDirectory
+        git checkout 'bdae0904c41a0ee2c5204d6449038d3b5d551726~1'
+        Pop-Location
+
         Write-Host "Bootstrapping vcpkg..."
-        & "$env:HOMEDRIVE\vcpkg\bootstrap-vcpkg.bat"
+        & "$vcpkgDirectory\bootstrap-vcpkg.bat"
         if ($LastExitCode)
         {
             Throw "Failed to bootstrap vcpkg with exit code $LastExitCode"
         }
         Write-Host "Installing vcpkg..."
-        & $env:HOMEDRIVE\\vcpkg\\vcpkg.exe integrate install
+        & $vcpkgDirectory\vcpkg.exe integrate install
```

Comment:
do you need double quote here?

Reply:
No, otherwise the original code won't have worked either.